### PR TITLE
[Enhancement] - add bigdl resnet-50 validation

### PR DESCRIPTION
**Pull request by wzhongyuan**
## What changes were proposed in this pull request?
Add validation job for resnet-50 pre-trained by BigDL
## How was this patch tested?
Unit test and example

### CODE REVIEW

**Comment by wzhongyuan** [comment]
Already using this branch to run the pre-trained bigdl model

Below is the log for accuracy 

2018-05-29 13:30:28 INFO  ModelValidator$:146 - Top1Accuracy is Accuracy(correct: 38055, count: 50000, accuracy: 0.7611)
2018-05-29 13:30:28 INFO  ModelValidator$:146 - Top5Accuracy is Accuracy(correct: 46399, count: 50000, accuracy: 0.92798)

**Comment by yiheng** [comment]
Looks good to me. Should we wait for inception v3 example code?

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/DatasetUtil.scala [hunk 1]
```diff
@@ -102,12 +103,26 @@ object ResNetPreprocessor {
...
-      BGRImgNormalizer(0.485, 0.456, 0.406, 0.229, 0.224, 0.225) -> BGRImgToSample()
-    transfomer(dataSet)
+    if (modelType == TorchModel) {
+      val dataSet = DataSet.SeqFileFolder.filesToRdd(path, sc, classNum = 1000)
+      val transfomer = BytesToBGRImg() ->
+        BGRImgCropper(cropWidth = imageSize, cropHeight = imageSize, CropCenter) ->
+        BGRImgNormalizer(0.485, 0.456, 0.406, 0.229, 0.224, 0.225) -> BGRImgToSample()
+      transfomer(dataSet)
+    } else if (modelType == BigDlModel) {
+      val data = DataSet.SeqFileFolder.filesToImageFrame(path, sc, 1000)
```
Comment (yiheng) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/DatasetUtil.scala]: different model different prerpocessing?

Comment (wzhongyuan) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/DatasetUtil.scala]: Above one is from Resnet, which is quite different

Comment (wzhongyuan) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/DatasetUtil.scala]: Above one is from Torch, which is quite different

**Comment by wzhongyuan** [comment]
PR validation passed http://mlt-bddl01.sh.intel.com:8080/job/BigDL-PRVN-lite/30/ 
Shall we merge it?
